docroot/themes/humsci/humsci_basic/src/js/media/video-with-caption.js

```diff
@@ -0,0 +1,16 @@
+// In order to style the figcaption, figure elements have a display: table.
+// This causes issues when there is a video in a figure because the video no longer
+// fills the entire space of the container.
+// This JS sets a width of 100% to figures that contain videos.
+const videos = document.querySelectorAll('.field-media-oembed-video');
```

Comment:
TODO for Lindsay: find a video to upload to ensure this works with other types of content

Reply:
I don't see an option to upload a video, only to link to an external source. I'll check with Alex at standup to confirm this is the case.

Reply:
Ok sounds good! Let's remember to ask if it's possible at standup, but not let it hold up this PR.